benchmarks/babelstream/src/babelStreamMainTest.cpp

```diff
@@ -57,13 +66,14 @@ struct InitKernel
     //! \tparam T The data type
     //! \param acc The accelerator to be executed on.
     //! \param a Pointer for vector a
-    //! \param initA the value to set all items in the vector
+    //! \param initialA the value to set all items in the vector a
+    //! \param initialB the value to set all items in the vector b
     template<typename TAcc, typename T>
-    ALPAKA_FN_ACC void operator()(TAcc const& acc, T* a, T* b, T* c, T initA) const
+    ALPAKA_FN_ACC void operator()(TAcc const& acc, T* a, T* b, T* c, T initialA, T initialB) const
     {
         auto const [i] = alpaka::getIdx<alpaka::Grid, alpaka::Threads>(acc);
-        a[i] = initA;
-        b[i] = static_cast<T>(0.0);
+        a[i] = initialA;
+        b[i] = initialB;
         c[i] = static_cast<T>(0.0);
```

Comment:
Why is this a compile-time constant? Compilers might do funny optimisations here if we give them too much of a hint.

Reply:
Won't make a difference i think, now changed and a new argument is used for C

Reply:
Thanks!

Reply:
I agree that it probably won't.